app/src/components/common/claim_dai_modal/index.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect } from 'react'
+import styled from 'styled-components'
+
+import { State, useXdaiBridge } from '../../../hooks/useXdaiBridge'
+import { formatBigNumber } from '../../../util/tools'
+import { Button } from '../../button/button'
+import { ButtonRound } from '../../button/button_round'
+import { ButtonType } from '../../button/button_styling_types'
+import { IconClose } from '../icons'
+import { IconArrowUp } from '../icons/IconArrowUp'
+import { DaiIcon } from '../icons/currencies'
+import { Spinner } from '../spinner'
+import { Modal, ModalBackground } from '../switch_network_modal'
+
+const ClaimAmount = styled.div`
+  margin-top: 32px;
+  font-size: 16px;
+  line-height: 18.75px;
+  color: ${props => props.theme.textfield.color};
+`
+const SecondaryText = styled.div`
+  margin-top: 12px;
+  color: ${props => props.theme.colors.textColor};
+  font-size: ${props => props.theme.fonts.defaultSize};
+  text-align: center;
+`
+const ClaimButton = styled(Button)`
+  margin-top: 32px;
+  font-weight: 500;
+`
+const CloseStyled = styled.div`
+  align-self: flex-end;
+  cursor: pointer;
+  margin-bottom: 32px;
+`
+const CloseButton = styled(ButtonRound)`
+  width: 252px;
+  margin-top: 32px;
+`
+const TransactionLink = styled.a`
+  margin-top: 16px;
+  color: ${({ theme }) => theme.colors.clickable};
+`
+const Reversed = styled.div`
+  transform: rotate(180deg);
+`
+
+export const ClaimDaiModal = (props: any) => {
```

Comment:
Sometimes when I refresh on xDAI the claim modal for mainnet will show up in the account dropdown. Potentially this might not happen on the dedicated xDai network url.

<img width="184" alt="Screen Shot 2021-02-03 at 11 39 12 am" src="https://user-images.githubusercontent.com/72715177/106682106-e8bd5280-6615-11eb-92f1-04f4df2a8c0c.png">

Reply:
Good catch i forgot to add checker for network for the dropdown item!